### DISCUSSION TITLE
AX9000硬路由挂载的，升级到最新的2.0.4-1还是不能使用，是这个方式不行了吗？

### DISCUSSION THREAD

**Discussion by GiveAShit** [messense/aliyundrive-webdav#755] [Q&A] [answered]
root@XiaoQiang:~# ps | grep aliyundrive-webdav
21551 root      1348 S    grep aliyundrive-webdav

这个意思是正常运行状态吧，但是指定的路由器端口8066还是打不开。

 curl -v http://192.168.31.1:8066
*   Trying 192.168.31.1:8066...
* connect to 192.168.31.1 port 8066 failed: Connection refused
* Failed to connect to 192.168.31.1 port 8066 after 10 ms: Couldn't connect to server
* Closing connection 0
curl: (7) Failed to connect to 192.168.31.1 port 8066 after 10 ms: Couldn't connect to server

**Comment by jahov0820** [comment]
和你一样是ax9000 日志报出来是token的问题 
Error: Invalid refresh token value found in --refresh-token argument

**Comment by jahov0820** [comment, marked as answer]
亲测使用二维码的方式获取的token是有效的, token的格式貌似已经变了 网页localstorage拿出来的是无效的
目前已经可以正常播放

**Reply by GiveAShit** [reply]
确实是token问题，已解决。
